decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -169,10 +178,30 @@ def profiling_gems
         copy_file "rack_profiler_initializer.rb", "config/initializers/rack_profiler.rb"
       end
 
+      def tweak_spring
```

Comment:
Another controversial decision: we could fully remove spring (one less dependency!), as according to the release notes newer hardware make this unnecessary in most cases. Ideally, we could make some performance testing to see if that's the case, but I guess it may depend on implementers' hardware too? 

In my head, leaving spring for now seemed like the less disrupting decision.